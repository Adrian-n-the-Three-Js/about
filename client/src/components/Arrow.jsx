// import React from 'react';
// import styled from 'styled-components';

// console.log('arrow');

// const Arrow = (props) => {

//   const ArrowButton = styled.button`
//     position: absolute;
//     width: 60px;
//     height: 60px;
//     cursor: pointer;
//     visibility: hidden
//     // align-self: flex-end;
//     // align-self: ${props => props.direction === 'right' ? 'flex-end' : 'flex:start'}
//     // top: 50%;
//     // background-color: #2c2c2c;
//     ${props.direction === 'right' ? 'right: 999px' : 'left: 43px'};
//     &:hover {
//       background-color: yellow;
//     }
// `;

//   return (

//     <ArrowButton className="arrow"
//       onClick={props.onClick}
//       // alt="navigation-arrow"
//       // src={props.direction === 'right' ? 'right-arrow.png' : 'left-arrow.png'}
//     >
//     {props.symbol}
//     </ArrowButton>

//   );
// };

// // const ArrowImage = styled.div`
// //   position: absolute;
// //   // display: flex;
// //   margin: auto;
// //   font-weight: bold;
// //   width: 60px;
// //   height: 60px;
// //   background-color: #2c2c2c;
// //   text-align: center;
// //   float: right;
// //   cursor: pointer;
// //   // bottom: 100px;
// //   // align-items: center;
// //   // justify-content: center;
// //   // right: ${(props) => (props.direction === 'right' ? '25px' : '0px')};
// //   // left: ${(props) => (props.direction === 'left' ? '25px' : '0px')};
// //   // ${(props) => (props.direction === 'right' ? 'color: red' : 'left: 25px')};
// //   // display: inline-block
// //   // padding: 5px;
// //   // border-radius: 3px 0px 0px 3px;
// //   // margin: -30px;
// //   // opacity: 50%;
// //   // &: focus {
// //   //   outline: 1;
// //   // }
// //   &:hover {
// //     background-color: yellow;
// //   }
// // `;

// export default Arrow;
