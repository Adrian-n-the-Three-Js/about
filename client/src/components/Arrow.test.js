import React from 'react';
import { mount, shallow, render } from 'enzyme';
import Arrow from './Arrow.jsx';

describe ('Arrow', () => {
  xit('should render correctly', () => {
    const component = shallow(<Arrow />);

    expect(component).toMatchSnapshot();
  });
});
