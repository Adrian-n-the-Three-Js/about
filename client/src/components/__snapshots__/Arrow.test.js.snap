// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow should render correctly 1`] = `<Arrow__ArrowImage />`;
